state/convert_test.go

```diff
@@ -88,24 +88,30 @@ func TestDeriveEventsHash(t *testing.T) {
 	client := spork.AccessNodes.Client()
 	for blockHeight := startBlockHeight; blockHeight < endBlockHeight; blockHeight++ {
 		block, err := client.BlockByHeight(ctx, blockHeight)
+		txns, err := client.TransactionsByBlockID(ctx, block.Id)
+		assert.NoError(t, err)
```

Comment:
nit, many of these `assert.NoError` should really be `require.NoError`. `require` aborts the test immediately if it's false, which is appropriate if the condition must be true for the rest of the test to be valid.